source/includes/answers.md

```diff
@@ -430,6 +430,670 @@ Parameter | Description
+      "followers": [],
+      "askers": [],
+      "comments": [],
+      "topics": [],
+      "score": 0,
+      "status": "published",
+      "counts": {
+        "asks": 1,
+        "downvotes": 0,
+        "upvotes": 0,
+        "edits": 1,
+        "comments": 0,
+        "hiddenAnswers": 0,
+        "answers": 1,
+        "views": 1,
+        "followers": 0
+      },
+      "type": "general",
+      "privacy": "public",
+      "anonymous": false,
+      "verified": false,
+      "locked": false,
+      "id": "5c7bfc7a157c2c34f735a53e"
+    },
+    "user": {
+      "_id": "5c7bf251157c2c34f735a53a",
+      "profile": {
+        "counts": {
+          "requests": 0,
+          "edits": 0,
+          "questionsFollowers": 0,
+          "questions": 2,
+          "comments": 0,
+          "answersWords": 3,
+          "answersVotes": 0,
+          "answersViews": 0,
+          "answers": 1,
+          "views": 0,
+          "followings": 0,
+          "followers": 0
+        },
+        "score": 10,
+        "verified": false,
+        "description": "",
+        "title": "",
+        "picture": "https://gravatar.com/avatar/b7711ecd91aebb5dd1b7c153dbfd92d0?s=200&d=retro",
+        "website": "",
+        "location": "",
+        "gender": "",
+        "name": "Admin",
+        "username": "admin"
+      },
+      "id": "5c7bf251157c2c34f735a53a",
+      "followed": false
+    },
+    "__v": 0,
+    "downvotes": [],
+    "upvotes": [],
+    "comments": [],
+    "rewards": [],
+    "images": [],
+    "links": [],
+    "score": 0,
+    "global": true,
+    "status": "published",
+    "media": [],
+    "counts": {
+      "reasks": 0,
+      "edits": 1,
+      "downvotes": 0,
+      "upvotes": 0,
+      "comments": 0,
+      "views": 0
+    },
+    "privacy": "public",
+    "anonymous": false,
+    "verified": false,
+    "summary": "The first answer",
+    "id": "5c7bfc9a157c2c34f735a540",
+    "upvoted": false,
+    "downvoted": false
+  }
+]
+```
+
+
+
+This endpoint retrieves all answers for a sepcific question using ID.
+
+### HTTP Request
+
+<code class="request">GET /api/v1/questions/:questionId/answers</code>
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+questionId | The ID of the question
+
+### Query Parameters
+
+Parameter | Default | Description
+--------- | ------- | -----------
+page | 1 | Intended page
+limit | 20 | Number of items per page
+sort | createdAt.desc | The field to sort on
+
+<!-- // TODO: -->
+<!-- 
+## Get All Answers for a Specific Topic
+
+```shell
+curl "https://community.tribe.so/api/v1/topics/5c7bfc7a157c2c34f735a53e/answers"
+  -H "Authorization: Bearer {access_token}"
+```
+
+
+This endpoint retrieves all answers for a sepcific topic using ID.
+
+### HTTP Request
+
+<code class="request">GET /api/v1/topics/:topicId/answers</code>
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+topicId | The ID of the topic -->
+
+## Get All Answers for a Specific User
```

Comment:
We already have this under users. It might not be as complete. We can complete what already exists under users.